Dockerfile

```diff
@@ -6,54 +6,58 @@ ENV LANG C.UTF-8
 # Manage i386 arch
 RUN dpkg --add-architecture i386
 
-# Install required packages (in sync with README.rst instructions
+# Install required packages (in sync with README.rst instructions)
 RUN apt-get update && apt-get install --no-install-recommends -y \
-        autogen \
-        automake \
-        bc \
-        bison \
-        build-essential \
-        check \
-        cmake \
-        curl \
-        cython \
-        debootstrap \
-        expect \
-        flex \
-        g++-multilib \
-        gettext \
-        git \
-        gperf \
-        imagemagick \
-        intltool \
-        libbz2-dev \
-        libc6-i386 \
-        libcppunit-dev \
-        libffi-dev \
-        libgc-dev \
-        libgmp3-dev \
-        libltdl-dev \
-        libmount-dev \
-        libncurses-dev \
-        libpcre3-dev \
-        libssl-dev \
-        libtool \
-        libunistring-dev \
-        lzip \
-        mercurial \
-        ncurses-dev \
-        php \
-        pkg-config \
-        python3 \
-        python3-distutils \
-        scons \
-        subversion \
-        swig \
-        unzip \
-        xmlto \
-        zlib1g-dev && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+		autogen \
+		automake \
+		bc \
+		bison \
+		build-essential \
+		check \
+		cmake \
+		curl \
+		cython \
+		debootstrap \
+		ed \
+		expect \
+		flex \
+		g++-multilib \
+		gawk \
+		gettext \
+		git \
+		gperf \
+		imagemagick \
+		intltool \
+		libbz2-dev \
+		libc6-i386 \
+		libcppunit-dev \
+		libffi-dev \
+		libgc-dev \
+		libgmp3-dev \
+		libltdl-dev \
+		libmount-dev \
+		libncurses-dev \
+		libpcre3-dev \
+		libssl-dev \
+		libtool \
+		libunistring-dev \
+		lzip \
+		mercurial \
+		ncurses-dev \
+		php \
+		pkg-config \
+		pngquant \
+		python3 python3-distutils \
```

Comment:
Can `python3-distutils` be on a separate line like previously and all others?

Reply:
yes it can, but as the command line is limitted to 1023 characters I wanted to save some of them.
It does not matter to use separate lines, we even could write all in one line as we do in README.rst to document the requirements to setup a virtual machine :smile:.